docs/en/integrations/data-ingestion/clickpipes/index.md

```diff
@@ -37,9 +38,9 @@ import WarpStreamSVG from "../../images/logos/warpstream.svg";
 
   ![Fill out connection details](./images/cp_step2.png)
 
-4a. __Optional for Schema Registry__: Specify the complete URL of your Schema Registry server along with the precise RESTful path to the ID representing your preferred schema document. This schema will serve as the validation benchmark for all messages from your topic, consequently blocking any messages that fail the validation against the designated schema. Additionally, the Schema Registry allows new schema retreival via JSON_SR messages. In scenarios where your JSON_SR message carries a Schema Registry ID differing from the current one, Clickpipes will fetch the corresponding schema and use it for future validation.
+4a. __Optional for Schema Registry__: If you are using a Kafka streaming pipe, specify the complete URL of your Schema Registry server along with the precise RESTful path to the ID representing your preferred schema document. This schema will serve as the validation benchmark for all messages from your topic, consequently blocking any messages that fail the validation against the designated schema. Additionally, the Schema Registry allows new schema retreival via JSON_SR messages. In scenarios where your JSON_SR message carries a Schema Registry ID differing from the current one, Clickpipes will fetch the corresponding schema and use it for future validation.
```

Comment:
>  In scenarios where your JSON_SR message carries a Schema Registry ID differing from the current one

It's valid for AvroConfluent as well. It's not related to the point of the current PR though